src/components/youtube.jsx

```diff
@@ -0,0 +1,16 @@
+
+
+export function YouTube({ src }) {
+  return (
+    <div className="mr-auto my-12">
+      <iframe
+        width="560"
+        height="315"
+        src={src}
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        referrerpolicy="strict-origin-when-cross-origin"
+        allowfullscreen />
```

Comment:
Seems like `allowfullscreen` didn't work for me.
MDN tells me it's supposed to be `allow="fullscreen"` now.